src/config/typeorm.config.ts

```diff
@@ -1,15 +1,15 @@
-import { TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { DataSource, DataSourceOptions } from 'typeorm';
 
-const typeOrmConfig: TypeOrmModuleOptions = {
+export const dataSourceOptions: DataSourceOptions = {
   type: 'postgres',
   host: process.env.DB_HOST || 'localhost',
   port: parseInt(process.env.DB_PORT) || 5432,
   username: process.env.DB_USERNAME,
   password: process.env.DB_PASSWORD,
   database: process.env.DB_NAME || 'elections_us',
   entities: [__dirname + '/../**/*.entity{.ts,.js}'],
-  migrations: [__dirname + '/../migrations/{.ts,.js}'],
-  synchronize: true,
+  migrations: [__dirname + '/../migrations/*{.ts,.js}'],
```

Comment:
Let's add `migrationsTableName: 'migration'` to track all migrations

Reply:
By default `migrationTableName === 'migrations'`